user-docs/pupil-capture.md

```diff
@@ -209,67 +224,190 @@ If you open up a session folder you will see a collection of video(s) and data f
 
 Click on the selector "Open Plugin" and select your plugin.
 
-### Pupil Groups
+#### Third-party plugins
+
+Third-party plugins can be loaded easily as well. In order for them to appear in
+the plugin list they need to be copied to the correct location, the `plugins`
+folder within your `capture_settings`.
```

Comment:
You can easily load third party plugins. Third party plugins will appear in the Pupil Capture or Pupil Player plugin list. Copy the plugin to the `plugins` folder within the `pupil_capture_settings` or `pupil_player_settings` folder.